docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,209 @@
+# Percona Monitoring and Management 2.38.0
+
+| **Release date:** | Jun 02, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### Important security updates
+
+**RHEL 9**
+
+As security is paramount to us, starting with PMM 2.38.0, we have laid a solid and modernized enterprise-grade foundation for PMM. We have migrated our base operating system on which we build our base image from Enterprise Linux 7 ( CentOS 7) to Enterprise Linux 9 (EL9). Thus starting with PMM 2.38.0, we have made PMM publicly available on EL9 (Oracle Linux 9) as an early preview.
+
+There are many benefits of this transition. The migration fixes a large number of CVEs and will see improved upstream response time to security-related issues. It also uses more modern libraries for our team to build on long with better compatibility with new technologies. 
+
+We will be rolling this out in phases. 
+
+The following users will be able to take advantage of this immediately:
+
+*New installations of PMM* 
+
+- New installations will pull down the EL9 container by default
+- Existing users that upgrade PMM using a full container replacement method will also pull the EL9 container
+- Fresh AMI installations
+- Fresh OVF installations
+
+Users who will need to wait a bit longer:
+
+- Users that exclusively upgrade through the UI’s “Upgrade Now” button including:
+-- Docker users that use UI upgrade
+-- AMI users
+-- OVF users
+
```

Comment:
We need here link or copy from Steve's blog post for the  workaround about migration to new image for this cases

Reply:
we need a link or instructions on how to do a migration for this group of people.

Reply:
@rnovikovP  Steve's blog is not yet published.

Reply:
Can we then embed the basic instructions for manual migration into this document

Reply:
tweaked blog post... https://docs.google.com/document/d/1M6tCSqoqxVkEg47vQutG1YH4Wv82sY6llh5Yd7mneAc/edit

review, bless and I'll get it published ASAP.